CHANGELOG.md

```diff
@@ -1,12 +1,24 @@
 ## Unreleased
 
+- The way contacts are represented changed. Refer to the documentation
+  of `parry::query::ContactManifold`, `parry::query::TrackedContact`
+  and `rapier::geometry::ContactManifoldData` and `rapier::geometry::ContactData` for details.
+
+### Breaking changes
+
+- `InteractionGroups` struct now contains `InteractionTestMode`.
+- `InteractionGroups` constructor now requires an `InteractionTestMode` parameter. If you want same behaviour as before, use `InteractionTestMode::AND` (eg. `InteractionGroups::new(Group::GROUP_1, Group::GROUP_1, InteractionTestMode::AND)`)
+- `CoefficientCombineRule::Min` - now makes sure it uses a non zero value as result by using `coeff1.min(coeff2).abs()`
```

Comment:
Thanks for filling the changelog ❤️. I'm happy with that description!

Previous entries of changelog don't have a particular "braking change" section and most end up in the "Modified" section, but that's details which can be edited on merge 👌 .

As it's not a trivial change, I'll probably want to add a link to that PR for additional context for the reader.

Reply:
Moved to modified.